# Generated by Django 4.2.5 on 2023-11-19 06:21

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('app', '0028_timeslot_trainer_alter_trainer_time_slots'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='timeslot',
            name='trainer',
        ),
        migrations.AddField(
            model_name='timeslot',
            name='trainers',
            field=models.ManyToManyField(related_name='trainer_time_slots', to='app.trainer'),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
        migrations.AlterField(
            model_name='trainer',
            name='time_slots',
            field=models.ManyToManyField(related_name='trainer_time_slots', to='app.timeslot'),
        ),
    ]
